modules/test/playwright/tests/frontend-js-spa-web/customExcludePathSPA.spec.ts

```diff
@@ -0,0 +1,59 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {isolatedLayoutTest} from '../../fixtures/isolatedLayoutTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {systemSettingsPageTest} from '../../fixtures/systemSettingsPageTest';
+import {waitForSuccessAlert} from '../../utils/waitForSuccessAlert';
+
+export const test = mergeTests(
+	isolatedLayoutTest(),
+	loginTest(),
+	systemSettingsPageTest
+);
+
+test('@LPD-26354 Custom Exclude Path', async ({
+	layout,
+	page,
+	systemSettingsPage,
+}) => {
+	await test.step('Exclude path in SPA Settings', async () => {
+		await systemSettingsPage.goToSystemSetting(
+			'Infrastructure',
+			'Frontend SPA Infrastructure'
+		);
+
+		const customExcludePath = page.getByLabel('Custom Excluded Paths', {
+			exact: true,
+		});
+
+		await customExcludePath.waitFor({state: 'visible'});
+		await customExcludePath.click();
+		await customExcludePath.fill(layout.friendlyURL);
+
+		const updateButton = page.getByRole('button', {
+			name: 'Update',
+		});
+
+		await updateButton.isVisible();
+		await updateButton.click();
+
+		await waitForSuccessAlert(page);
+	});
+
+	await test.step('Go to page and check SPA', async () => {
+		await page.goto(layout.friendlyURL);
+
+		await page.reload();
+
+		// @ts-ignore
+
+		const liferaySPAOutput = await page.evaluate(() => Liferay.SPA);
+
+		expect(liferaySPAOutput).toEqual(undefined);
```

Comment:
To improve the test and validate this logic, could we add a step on start, that makes the same check and asserts something like 
```js
expect(output).not.toEqual(undefined);
```

Reply:
Added the check, thank you!